services/template.py

```diff
@@ -0,0 +1,19 @@
+from jinja2 import Environment, PackageLoader, StrictUndefined, select_autoescape
+
+
+def get_template_service():
```

Comment:
I think this is unnecessary. We can very well just instantiate `TemplateService()` directly